verification/tpm.go

```diff
@@ -0,0 +1,268 @@
+// Licensed under the Apache-2.0 license
+
+package verification
+
+import (
+	"crypto/ecdsa"
+	"crypto/sha256"
+	"crypto/sha512"
+	"crypto/x509"
+	"encoding/binary"
+	"flag"
+	"io"
+	"math"
+	"math/big"
+	"strings"
+	"testing"
+
+	"github.com/google/go-tpm-tools/client"
+	"github.com/google/go-tpm/legacy/tpm2"
+	"github.com/google/go-tpm/tpmutil"
+)
+
+var (
+	tpmPath                    = flag.String("tpm-path", "/dev/tpm0", "Path to the TPM device (character device or a Unix socket).")
+	tpmPolicySigningValidation = flag.String("tpm-policy-signing-validation", "disabled", "Validation of TPM policy signing with Caliptra DPE commands is enabled/disabled. By default, this is disabled.")
```

Comment:
Why would a caller pass `disabled` for this flag?

Reply:
Unable to initialize the tpm simulator in pipeline as some packages are missing in azure ubuntu. To skip pipeline failure. this is added.

Reply:
This issue is fixed